examples/petsc/setupsphere.h

```diff
@@ -103,8 +103,10 @@ static const char *const bpTypes[] = {"bp1","bp2","bp3","bp4","bp5","bp6",
 typedef struct {
   CeedInt ncompu, qdatasize, qextra;
   CeedQFunctionUser setupgeo, setuprhs, apply, error, ident;
+  // *INDENT-OFF*
   const char *setupgeofname, *setuprhsfname, *applyfname, *errorfname,
-        *identfname;
+             *identfname;
```

Comment:
Thanks @jeremylt, I think you may actually drop this. It appears not to be used

Reply:
Right.